src/ShadowDOM/wrappers/Range.js

```diff
@@ -24,6 +24,13 @@
   var ShadowRoot = scope.wrappers.ShadowRoot;
 
   function getHost(node) {
+    var parent = node;
+    while (parent = parent.parentNode) {
+      if (parent.shadowRoot) {
```

Comment:
Can you explain and document why the host should be `null` if any ancestor has a `shadowRoot`?